### PR TITLE
Add isPublic operator

### PR DESCRIPTION
**Pull request by arj03**
To complement isPrivate

### CODE REVIEW

File: operators/index.js [hunk 1]
```diff
@@ -83,6 +84,10 @@ function isPrivate() {
   return equal(seekPrivate, bTrue, { indexType: 'meta_private' })
 }
 
+function isNotPrivate() {
+  return equal(seekMeta, undefined, { indexType: 'meta_private' })
```
Comment (staltz) [inline operators/index.js:88]: Shouldn't the indexType be `'meta'`?

**Review by staltz — CHANGES_REQUESTED** [review]

File: operators/index.js (hunk 1 above)
Comment (arj03) [reply operators/index.js]: Sure, I guess that is a bit cleaner

File: test/operators.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const {
   and,
   type,
   isPrivate,
+  isNotPrivate,
```
Comment (mixmix) [inline test/operators.js:13]: Why not isPublic?

Comment (arj03) [reply test/operators.js:13]: A very good suggestion :D Thanks

**Review by staltz — APPROVED** [review]
Thanks! `isPublic` is good too

**Pull request merged**